dev/drivers/scripts/analyses/stats/jevs_analyses_urma_grid2obs_stats.sh

```diff
@@ -6,12 +6,12 @@
 #PBS -q "dev"
 #PBS -A VERF-DEV
 #PBS -l walltime=00:30:00
-#PBS -l select=1:ncpus=1:mem=2GB
+#PBS -l select=1:ncpus=1:mem=10GB
 #PBS -l debug=true
 
 export model=evs
 
-export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS_analyses_EE2/EVS
```

Comment:
Can you revert this HOMEevs definition? ... and all the rest of the drivers

Reply:
Also above.